vico/compose/src/main/java/com/patrykandpatrick/vico/compose/cartesian/CartesianChartHost.kt

```diff
@@ -146,7 +146,7 @@ internal fun CartesianChartHostImpl(
       ranges = ranges,
       scrollEnabled = scrollState.scrollEnabled,
       zoomEnabled = scrollState.scrollEnabled && zoomState.zoomEnabled,
-      layerPadding = chart.layerPadding,
+      layerPadding = chart.getLayerPadding.invoke(model.extraStore),
```

Comment:
Please call the lambda normally. Also, the result should be cached until `ExtraStore` changes.
```suggestion
      layerPadding = remember(chart, chart.getLayerPadding, model.extraStore) { chart.getLayerPadding(model.extraStore) },
```

Reply:
Good call! I'm just curious, are we sure we need all of those keys? Particularly the `chart` one. I did add it, but just calling out perhaps only `layerPadding` and `extraStore` are necessary?

Reply:
>Particularly the chart one.

Yes, it's not needed, actually. There was a mistake in the suggestion.